static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -90,22 +132,50 @@
 
   "nodes": {
     "title": "Visor Liste",
+    "dmsg-title": "DMSG",
+    "update-all": "Alle Visor aktualisieren",
+    "hypervisor": "Hypervisor",
     "state": "Status",
+    "state-tooltip": "Aktueller Status",
     "label": "Bezeichnung",
     "key": "Schlüssel",
+    "dmsg-server": "DMSG Server",
+    "ping": "Ping",
+    "hypervisor-info": "Dieser Visor ist der aktuelle Hypervisor.",
+    "copy-key": "Schlüssel kopieren",
+    "copy-dmsg": "DMSG Server Schlüssel kopieren",
+    "copy-data": "Daten kopieren",
     "view-node": "Visor betrachten",
     "delete-node": "Visor löschen",
+    "delete-all-offline": "Alle offline Visor löschen",
     "error-load": "Beim Aktualisieren der Visor-Liste ist ein Fehler aufgetreten.",
     "empty": "Es ist kein Visor zu diesem Hypervisor verbunden.",
+    "empty-with-filter": "Kein Visor erfüllt die gewählten Filterkriterien",
```

Comment:
Punctuation at the end of the sentence.

Reply:
Edited. Should we edit punctuation on all sentences? I think there are a few without at the moment.